pom.xml

```diff
@@ -171,7 +171,7 @@
         <dependency>
             <groupId>fr.acinq.bitcoin</groupId>
             <artifactId>bitcoin-kmp-jvm</artifactId>
-            <version>0.13.0</version>
+            <version>0.14.0-SNAPSHOT</version>
```

Comment:
You can now update `bitcoin-kmp` to 0.14.0 and `secp256k1-kmp` to 0.11.0

Reply:
Done in https://github.com/ACINQ/bitcoin-lib/pull/76/commits/0097a4467a4a41063e65ef300728035b087938e7

Reply:
Ah I'm missing the secp update, hold on.

Reply:
Done!